components/Frame/Header.js

```diff
@@ -366,31 +368,41 @@ class Header extends Component {
               height={this.state.mobile ? HEADER_HEIGHT_MOBILE : HEADER_HEIGHT}
             />
           )}
-          <Popover expanded={!!expand} inNativeApp={inNativeApp} me={me} isMobile={mobile}>
-            <NavPopover
-              me={me}
-              url={url}
-              inNativeApp={inNativeApp}
-              closeHandler={this.close}
-            />
-          </Popover>
         </div>
-        {!inNativeApp && me && opaque && (
-          <NavBar
+        {isMember && opaque && (
+          <Fragment>
+            <hr
+              {...styles.stickyWithFallback}
+              {...styles.hr}
+              {...styles.hrThin} />
+            <NavBar fixed={withoutSticky} url={url} />
+          </Fragment>
+        )}
+        <hr
```

Comment:
Edge case which is probably a side effect of using a HR:
On an article with format (i.e. a thick colored HR) when you zoom in on Chrome (e.g. 150%) the vertical position of the HR is slightly off:
<img width="1181" alt="screen shot 2018-08-23 at 11 21 46" src="https://user-images.githubusercontent.com/23520051/44517291-9ac36700-a6c7-11e8-9183-e8edf8b85c9e.png">

Reply:
✅ added a safety pixel to prevent shine though when zooming